src/cloud/docker/docker-containers-service.md

```diff
@@ -170,13 +170,26 @@ To mount custom NGINX configuration file using volumes:
 
 1. To mount the custom NGINX configuration to the Web container, add the volume configuration to the `docker-compose.override.yml` file.
 
-   ```conf
-   web:
-    volumes:
-      ./.docker/nginx/etc/nginx.conf:/etc/nginx/nginx.conf
-      ./.docker/nginx/etc/vhost.conf:/etc/nginx/conf.d/default.conf
+   ```yaml
+   services:
+     web:
+      volumes:
```

Comment:
looks like some indention problems in this yaml

Reply:
Not sure, why it's different for yaml.

Fixed